llvm/test/tools/sycl-post-link/device-code-split/per-aspect-split-2.ll

```diff
@@ -5,11 +5,11 @@
 ; RUN: FileCheck %s -input-file=%t.table --check-prefix CHECK-TABLE
```

Comment:
Should we add a test with aspects and large GRF to lock down the behavior?

Reply:
The thing is that as we add more features, they will affect existing tests anyway, so I would prefer to leave this as-is to have an isolated test for a piece of the implementation